device-management-endpoint/src/main.rs

```diff
@@ -0,0 +1,103 @@
+use drogue_cloud_database_common::database;
+use drogue_cloud_database_common::models;
+
+use actix_web::{delete, http::header, post, web, App, HttpResponse, HttpServer};
+
+use actix_web::web::Buf;
+use dotenv::dotenv;
+use envconfig::Envconfig;
+
+#[post("/device/{device_id}")]
+async fn create_device(
+    data: web::Data<WebData>,
+    web::Path(device_id): web::Path<String>,
+    mut body: web::Payload,
+) -> Result<HttpResponse, actix_web::Error> {
+    log::info!("Creating device: '{}'", device_id);
+
+    let mut bytes = web::BytesMut::new();
+    while let Some(item) = body.next().await {
+        bytes.extend_from_slice(&item?);
+    }
```

Comment:
This doesn't compile even if it's a direct copypaste from : https://github.com/drogue-iot/drogue-cloud/blob/edfca9cccb06cd41a36c3286842d5d2fddb4c3cd/http-endpoint/src/main.rs#L55

I am using the same version of `actix-web` in cargo.tml as the http endpoint so i don't understand the issue..

```
error[E0599]: no method named `next` found for struct `actix_web::web::Payload` in the current scope
  --> device-management-endpoint/src/main.rs:19:33
   |
19 |     while let Some(item) = body.next().await {
   |                                 ^^^^ method not found in `actix_web::web::Payload`

```